src/components/DuplicateGroup/DuplicateGroup.tsx

```diff
@@ -7,15 +8,19 @@ import './DuplicateGroup.scss';
 
 interface Props {
   onClick?: VoidFunction;
+  hasDeleteButton?: boolean;
+  className?: string;
 }
 
-export const DuplicateGroup: FC<Props> = memo(({ onClick }) => (
-  <div className="duplicate-group">
+export const DuplicateGroup: FC<Props> = memo(({ onClick, hasDeleteButton = true, className }) => (
+  <div className={classNames(['duplicate-group', className])}>
     <Button data-testid="id-duplicate-group" type={ButtonType.Icon} onClick={onClick}>
       <Plus16 />
     </Button>
-    <Button data-testid="id-delete-duplicate-group" type={ButtonType.Icon} disabled={IS_DISABLED_FOR_ALPHA}>
-      <Trash16 />
-    </Button>
+    {hasDeleteButton && (
+      <Button data-testid="id-delete-duplicate-group" type={ButtonType.Icon} disabled={IS_DISABLED_FOR_ALPHA}>
```

Comment:
do we still need IS_DISABLED_FOR_ALPHA flag?

Reply:
I don't think so.
And this is out of the scope of the current task. I think, this button will be enabled in the scope of implementation of the required functionality.